pygmt/tests/test_datasets_earth_magnetic_anomaly.py

```diff
@@ -31,7 +31,9 @@ def test_earth_mag_01d():
     """
     Test some properties of the magnetic anomaly 01d data.
     """
-    data = load_earth_magnetic_anomaly(resolution="01d", registration="gridline")
+    data = load_earth_magnetic_anomaly(
+        resolution="01d", registration="gridline", mag4km=False
+    )
```

Comment:
I prefer the old test without `mag4km=False`, because it also checks if the default dataset type (i.e., `earth_mag`) is changed.